cmd/stunstamp/stunstamp.go

```diff
@@ -34,6 +35,24 @@ import (
 	"tailscale.com/tailcfg"
 )
 
+type dstPortsFlags []int
+
+func (d *dstPortsFlags) String() string {
+	return fmt.Sprintf("%v", *d)
+}
+
+func (d *dstPortsFlags) Set(value string) error {
+	i, err := strconv.Atoi(value)
```

Comment:
You can also use `strconv.ParseInt` which will handle the bounds check for you, if you like:

```go
i, err := strconv.ParseUint(value, 10, 16)
if err != nil {
   return err // includes if i is out of bounds
}
*d = append(*d, int(i))
return nil
```

But also: While a "repeatable" flag works, it's an unusual pattern for a Go flag, so I'd suggest taking it as a comma-separated string instead.

Reply:
updated in https://github.com/tailscale/tailscale/pull/12356/commits/58863f931f081c5a5d6ff886660bd66d76a6ecc0